### ISSUE TITLE
 calc -h kills a-shell

### ISSUE THREAD

**Issue by ifuchs** [holzschu/a-shell#113]
Subject says it all.

**Comment by personalizedrefrigerator** [comment]
Note: Running `calc -h > help.txt` still kills `a-Shell`.

Running `cat help.txt` produces
```

Usage: calc [options] '<expression>'

Options:
    -h|--help                  Display this usage
   -d|--debug                 Debug mode
   -i|--infix                 Infix expression (default)
   -p|--postfix               Postfix expression
```

**Comment by holzschu** [comment]
Thank you for signalling it. This issue is fixed in the latest release.

**Comment by holzschu** [comment]
I haven't looked at "calc -i", though.

**Comment by holzschu** [comment]
According to `calc -h`, `calc -i` is "infix expression", not "interactive mode".

**Comment by ifuchs** [comment]
I just read it in the man page (perhaps there is more than one version?)

From the Man page:



       *-i*     Become  interactive  if  possible.  This flag will
cause *calc* to
              drop into interactive mode after the *calc**_**cmd*
arguments  on  the
              command  line  are evaluated.  Without this flag, *calc* will exit
              after they are evaluated.

              For example:

                   calc 2+5

              will print the value 7 and exit whereas:

                   calc -i 2+5

              will print the value 7 and prompt the user for  more  *calc*  com-
              mands.

**Comment by holzschu** [comment]
Yes, on a-Shell it's the calc utility that comes with taskwarrior: https://taskwarrior.org/docs/commands/calc.html#utility
Whereas this man page is for the calc command: http://www.isthe.com/chongo/tech/comp/calc/

For computations on a-Shell, there is also `bc`.
